pkg/podvolume/backupper_factory.go

```diff
@@ -41,45 +40,46 @@ type BackupperFactory interface {
 func NewBackupperFactory(
 	repoLocker *repository.RepoLocker,
 	repoEnsurer *repository.Ensurer,
-	veleroClient clientset.Interface,
 	pvcClient corev1client.PersistentVolumeClaimsGetter,
 	pvClient corev1client.PersistentVolumesGetter,
 	podClient corev1client.PodsGetter,
+	crClient ctrlclient.Client,
+	crWatchClient ctrlclient.WithWatch,
 	log logrus.FieldLogger,
 ) BackupperFactory {
 	return &backupperFactory{
-		repoLocker:   repoLocker,
-		repoEnsurer:  repoEnsurer,
-		veleroClient: veleroClient,
-		pvcClient:    pvcClient,
-		pvClient:     pvClient,
-		podClient:    podClient,
-		log:          log,
+		repoLocker:    repoLocker,
+		repoEnsurer:   repoEnsurer,
+		pvcClient:     pvcClient,
+		pvClient:      pvClient,
+		podClient:     podClient,
+		crClient:      crClient,
+		crWatchClient: crWatchClient,
+		log:           log,
 	}
 }
 
 type backupperFactory struct {
-	repoLocker   *repository.RepoLocker
-	repoEnsurer  *repository.Ensurer
-	veleroClient clientset.Interface
-	pvcClient    corev1client.PersistentVolumeClaimsGetter
-	pvClient     corev1client.PersistentVolumesGetter
-	podClient    corev1client.PodsGetter
-	log          logrus.FieldLogger
+	repoLocker    *repository.RepoLocker
+	repoEnsurer   *repository.Ensurer
+	pvcClient     corev1client.PersistentVolumeClaimsGetter
+	pvClient      corev1client.PersistentVolumesGetter
+	podClient     corev1client.PodsGetter
```

Comment:
Can we use the controller-runtime client instead for `PersistentVolumeClaimsGetter`, `PersistentVolumesGetter` and `PodsGetter`?

Reply:
Modified.